package-lock.json

```diff
@@ -1,16 +1,285 @@
 {
-  "name": "@metamask/swappable-obj-proxy",
+  "name": "swappable-obj-proxy",
   "version": "1.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Hmm.... looks like this was generated with an older version of npm?

Reply:
This got reverted by mistake. But this also isn't right on `master` either. I've submitted a new PR to fix it ahead of this PR: https://github.com/MetaMask/swappable-obj-proxy/pull/20

Reply:
Okay, these changes should have never been there in the first place on this branch. They've now been removed.